index.ts

```diff
@@ -429,6 +436,97 @@ function manifestExists(): boolean {
   return fs.existsSync(PROJECT_MANIFEST_FULLNAME);
 }
 
+/**
+ * Recursively finds all files that are part of the current project, and those that are ignored
+ * by .claspignore and calls the passed callback function with the file lists.
+ * @param {string} rootDir The project's root directory
+ * @param {FilesCallBack} callback The callback will be called with the following paramters
+ * error: Error if there's an error, otherwise null
+ * result: string[][], List of two lists of strings, ie. [nonIgnoredFilePaths,ignoredFilePaths]
+ * files?: Array<AppsScriptFile|undefined> Array of AppsScriptFile objects used by clasp push
+ */
+function getProjectFiles(rootDir: string, callback: FilesCallback): any {
```

Comment:
Don't use `any`. Find out the types of the return values.

Reply:
ACK.